src/OpenLoco/src/Environment.cpp

```diff
@@ -57,6 +57,7 @@ namespace OpenLoco::Environment
             "C:/Program Files (x86)/Atari/Locomotion",
             "C:/GOG Games/Chris Sawyer's Locomotion",
             "C:/GOG Games/Locomotion",
+            "."
```

Comment:
I know this works but this is not what this table is for. `.` is not a known path for the game.